data/Tejasp2003.json

```diff
@@ -9,12 +9,12 @@
 		},
 		{
 			"name": "Follow me on Github",
-			"url": "https://twitter.com/Tweetsoftejas",
+			"url": "https://twitter.com/Tejasp2003",
```

Comment:
Same here you have given url for twitter instead of GitHub

Reply:
Sorry! I have made corrections now